src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,18 @@
+import MainContainer from 'components/MainContainer';
+import { StyledFotter, StyledText } from './Footer.styled';
+
+const Footer = () => {
+  return (
+    <StyledFotter>
+      <MainContainer>
+        <StyledText>Join us on our site </StyledText>
+        <a href="https://chingu-voyages.github.io/v44-tier2-team-15/">
+          Here
+          {/* <img src={logo} alt="Logo" width="76" height="22" /> */}
```

Comment:
- [ ] I kindly suggest that you delete all unused pieces of code left as comments.  Commented code left as comments make the code base messy 😄

Reply:
Yes, i will delete it